apps/shopify/src/constants.js

```diff
@@ -2,6 +2,7 @@
 // returned with this unfortunate title, and there is no other way to check
 // whether the returned variant is the default one or not
 export const DEFAULT_SHOPIFY_VARIANT_TITLE = 'Default Title';
+export const DEFAULT_SHOPIFY_API_VERSION = '2022-04'
```

Comment:
I have seen this in some urls, but I don't see this passed in the Shopify client constructor. Isn't it needed there?

Reply:
no it is not needed in constructor as changing the version (2.15.1) in package.json will automatically pick the latest stable version